LeaveManagement/LeaveManagement/src/test/java/com/example/LeaveManagement/LeaveManagementApplicationTests.java

```diff
@@ -0,0 +1,5 @@
+package com.example.leavemanagement;
+import org.springframework.boot.test.context.SpringBootTest;
+@SpringBootTest
+class LeaveManagementApplicationTests {
```

Comment:
Ensure the unit test coverage is addressed.

Reply:
Addressing the PR comments provided above, I have planned to improve the unit test coverage.